app/views/home/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1><%= t :greetings %></h1>
+<h4>Default: <%= l Date.today %></h4>
+<h4>Short: <%= l Date.today, format :short %> </h4>
```

Comment:
use i18n for defalut and short

Reply:
I'm sorry, but I found out on ruby on rails guides, i18n for date is same as the code above. Did I miss something?